doc/release-notes-16807.md

```diff
@@ -1,6 +1,8 @@
 Updated RPCs
 ------------
 
-- The `validateaddress` RPC now optionally returns an `error_locations` array, with the indices of
-invalid characters in the address. For example, this will return the locations of up to two Bech32
-errors.
\ No newline at end of file
+- The `validateaddress` RPC now returns an `error_locations` array for invalid
+addresses, with the indices of invalid character locations in the address (if
+known). For example, this will return the locations of up to two Bech32 errors.
```

Comment:
Nit: there is really no way to know these are the *actual* error locations. They're a guess, except under the assumption that no more than 2 substitution errors have been made, in which case they're guaranteed to be correct.